lib/generators/templates/public.yml

```diff
@@ -0,0 +1,44 @@
+default: &default
+
+  app_options:
+    version: 0.1.0
+    # Header <h1>
+    project_name: Sample Template
+    # <title>, meta application-name
+    project_shortname: Sample
+    # Header <h2>
+    project_subtitle: Template Subtitle
+
+  api_options:
+    # only grab letters by default
```

Comment:
We should probably change these to not be defaults that fit a specific app, as that will result in headaches with almost every new app. Similarly maybe we should set the default time period to cover from further back until present day or some date in the future so we don't accidentally leave out results by default in new apps?

Reply:
Good idea, I left the filters in but commented out as an example for the future

Reply:
Magnifico